api/handlers_test.go

```diff
@@ -7,22 +7,31 @@ import (
 	"io"
 	"net/http"
 	"net/http/httptest"
+	"os"
+	"runtime"
 	"sync"
 	"testing"
 	"time"
 
+	"github.com/Sirupsen/logrus"
 	"github.com/dcos/dcos-diagnostics/config"
 	"github.com/gorilla/mux"
 	assertPackage "github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/suite"
 )
 
+var DiagnosticsBundleDir = "/tmp/snapshot-test"
 var testCfg *config.Config
 
 func init() {
+	if runtime.GOOS == "windows" {
+		DiagnosticsBundleDir = os.Getenv("SYSTEMDRIVE") + "\\tmp\\snapshot-test"
+	}
+	logrus.Infof("DiagnosticsBundleDir=%v", DiagnosticsBundleDir)
```

Comment:
ideally we should not be using logrus in tests

Reply:
That was the only message that's using logrus and it's not an essential message. I will remove them from this test file to make it cleaner.